index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <header id="header">
+      <img
+        src="https://www.freepnglogos.com/uploads/company-logo-png/company-logo-transparent-png-19.png"
+        id="header-img"
+        alt="Logo"
+      />
+      <nav id="nav-bar">
+        <a href="#product-section" class="nav-link">Our Product</a>
+        <a href="#about-section" class="nav-link">About us</a>
+        <a href="#contact-section" class="nav-link">Contact us</a>
+      </nav>
+    </header>
+
+    <div id="product-section">
+      <iframe
+        id="video"
+        width="560"
+        height="315"
```

Comment:
Kindly avoid using inline styles. I think it would be better if you keep your style definition in a separate CSS file. This helps keep your code clean.